hedyorg/hedy
# [TESTING] Expands on partly testing to include hedy.app and /tests folder

**Description**
In this PR we further improve the testing flow to make sure GitHub performs some smart testing to decrease the time before a PR is validated. We add `hedy.app` and the `tests` folder to the changes that will trigger all tests and we also check if the `force_tests` label is active on the PR, if so: run all tests as well.

**Fixes**
This PR fixes #3607

**How to test**
This should be fine as long as the tests run. Once the label is implemented, verify this as well.
